lib/collection.js

```diff
@@ -277,12 +277,13 @@ const DEPRECATED_FIND_OPTIONS = ['maxScan', 'fields', 'snapshot'];
  * @param {number} [options.maxScan] DEPRECATED: Limit the number of items to scan.
  * @param {number} [options.min] Set index bounds.
  * @param {number} [options.max] Set index bounds.
- * @param {boolean} [options.showDiskLoc] Show disk location of results.
+ * @param {boolean} [options.showDiskLoc=false] DEPRECATED. Show disk location of results.
+ * @param {boolean} [options.showRecordId] Return the record identifier for each document.
  * @param {string} [options.comment] You can put a $comment field on a query to make looking in the profiler logs simpler.
- * @param {boolean} [options.raw=false] Return document results as raw BSON buffers.
- * @param {boolean} [options.promoteLongs=true] Promotes Long values to number if they fit inside the 53 bits resolution.
```

Comment:
we need to be _very_ careful here. I believe these _are_ indeed properly documenting defaults, see [here](https://github.com/mongodb/js-bson/blob/master/lib/parser/deserializer.js#L80-L82)